base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1129,5 +1129,22 @@
 				</macros>
 			</value>
 		</entry>
+        <entry>
+			<key>Lakeshore 460</key>
+			<value>
+			    <categories>
+					<category>Monitors</category>
```

Comment:
Is this the most appropriate category for this device?  "Monitors" is used for neutron monitors in the beam.

Reply:
Changed to 'gaussmeter'